instrumentation/opentelemetry-instrumentation-jinja2/src/opentelemetry/instrumentation/jinja2/package.py

```diff
@@ -13,4 +13,4 @@
 # limitations under the License.
 
 
-_instruments = ("jinja2~=2.7",)
+_instruments = ("jinja2>=2.7",)
```

Comment:
Can we set it to `>=2.7,<4.0` so we don't automatically support a future version that might not work (4.0)?

Reply:
Done. Thank you.